mir_eval/segment.py

```diff
@@ -1227,71 +1215,39 @@ def evaluate(ref_intervals, ref_labels, est_intervals, est_labels, **kwargs):
     # Now compute all the metrics
     scores = collections.OrderedDict()
 
-    # Boundary detection
+    # Metrics for Intervals without structure labels section:
+    trim = kwargs.get('trim')
```

Comment:
This is a little unclear, since it's by default going to give `trim is None` rather than `trim is False`.  The behavior will be the same (at least until we add type annotations), but it would be better to explicitly populate a boolean.